watertap/flowsheets/full_water_resource_recovery_facility/test/test_BSM2_P_extension.py

```diff
@@ -54,61 +55,61 @@ def test_solve(self, system_frame):
         m = system_frame
 
         assert value(m.fs.Treated.properties[0].flow_vol) == pytest.approx(
-            0.2422, rel=1e-3
+            0.24219, rel=1e-3
         )
         assert value(m.fs.Treated.properties[0].conc_mass_comp["S_A"]) == pytest.approx(
-            4.453e-06, rel=1e-3
+            6.4300e-07, rel=1e-3
```

Comment:
I this meant to be a ~zero value? Wondering if this should be changed to absolute tolerance or not.

Reply:
Either way, for values this small you need to be using absolute tolerances (unless the model is really well scaled), as otherwise they are below solver tolerance and thus only accurate to maybe 1e-6 (absolute).

Reply:
@luohezhiming can you change this to absolute tolerance before I merge?